.github/workflows/self-test.yml

```diff
@@ -14,18 +14,19 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Statick
-        uses: sscpac/statick-action@v0.0.2
+        uses: gregtkogut/statick-action@add-timings-information
```

Comment:
- [ ] Change this line, and the below two similar lines to the expected release version before merging.